dhis-2/dhis-support/dhis-support-cache-invalidation/pom.xml

```diff
@@ -43,6 +43,11 @@
       <groupId>io.debezium</groupId>
       <artifactId>debezium-embedded</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.debezium</groupId>
+      <artifactId>debezium-connector-postgres</artifactId>
+      <scope>test</scope>
```

Comment:
If the scope is test the library will not be available when running the application.
Does it make sense?

If it needs to be available at runtime but it is not directly used, we need to add it as an exception in `maven-dependency-plugin` as a `ignoredNonTestScopedDependency`

Reply:
@enricocolasante Thanks, good catch. I fixed the build error by adding scope test but it broke the running. I've now done as you suggest and it runs.